ui/raidboss/data/03-hw/deepdungeon/the_palace_of_the_dead_general.ts

```diff
@@ -0,0 +1,184 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// Triggers applicable to all Palace of the Dead floors.
+
+export type Data = RaidbossData;
+
+// pomanders: https://xivapi.com/deepdungeonItem?pretty=true
+const pomanders = [
+  'Safety',
+  'Sight',
+  'Strength',
+  'Steel',
+  'Affluence',
+  'Flight',
+  'Alteration',
+  'Purity',
+  'Fortune',
+  'Witching',
+  'Serenity',
+  'Rage',
+  'Lust',
+  'Intuition',
+  'Raising',
+  'Resolution',
+  'Frailty',
+  'Concealment',
+  'Petrification',
+];
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: [
+    ZoneId.ThePalaceOfTheDeadFloors1_10,
+    ZoneId.ThePalaceOfTheDeadFloors11_20,
+    ZoneId.ThePalaceOfTheDeadFloors21_30,
+    ZoneId.ThePalaceOfTheDeadFloors31_40,
+    ZoneId.ThePalaceOfTheDeadFloors41_50,
+    ZoneId.ThePalaceOfTheDeadFloors51_60,
+    ZoneId.ThePalaceOfTheDeadFloors61_70,
+    ZoneId.ThePalaceOfTheDeadFloors71_80,
+    ZoneId.ThePalaceOfTheDeadFloors81_90,
+    ZoneId.ThePalaceOfTheDeadFloors91_100,
+    ZoneId.ThePalaceOfTheDeadFloors101_110,
+    ZoneId.ThePalaceOfTheDeadFloors111_120,
+    ZoneId.ThePalaceOfTheDeadFloors121_130,
+    ZoneId.ThePalaceOfTheDeadFloors131_140,
+    ZoneId.ThePalaceOfTheDeadFloors141_150,
+    ZoneId.ThePalaceOfTheDeadFloors151_160,
+    ZoneId.ThePalaceOfTheDeadFloors161_170,
+    ZoneId.ThePalaceOfTheDeadFloors171_180,
+    ZoneId.ThePalaceOfTheDeadFloors181_190,
+    ZoneId.ThePalaceOfTheDeadFloors191_200,
+  ],
+
+  triggers: [
+    // ---------------- Mimics ----------------
+    {
+      id: 'PotD General Mimic Spawn',
+      // 2566 = Mimic
+      // TODO: some Mimics may spawn after transference between floors and get called early before being found
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '2566', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Mimic spawned!',
+        },
+      },
+    },
+    {
+      id: 'PotD General Mimic Infatuation',
+      // gives Accursed Pox (43F) if not interrupted
+      type: 'StartsUsing',
+      netRegex: { id: '18FD', source: 'Mimic' },
```

Comment:
Is there a different ability id for the uninterruptible versions on later floors?